node/packages/sdk-schema/.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const path = require('path');
+
+const projectDir = path.resolve(__dirname, '../..');
+
+module.exports = {
+  extends: path.resolve(projectDir, '.eslintrc.js'),
+  rules: {
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        devDependencies: ['**/scripts/**', '**/test/**', 'prettier.config.js'],
+        packageDir: [projectDir, path.resolve(projectDir, 'packages/sdk-schema')],
+      },
+    ],
+  },
+  overrides: [
+    {
+      files: ['test/**/*.js'],
+      rules: {
+        'no-loop-func': 'off',
```

Comment:
We need it for `sdk-schema` since the test imports the built files, which do not exist at the point in our workflows when we lint. I don't want us to have to build `sdk-schema` every time the top level lint runs but we would need to do that if we don't have this.

Reply:
Here I was referring strictly to `'no-loop-func': 'off'` which I think is just result of copy/paste (?), and can be remove

Otherwise definitely we need `.eslintrc.js` here. I agree